django/db/backends/oracle/base.py

```diff
@@ -50,14 +50,14 @@ def _setup_environment(environ):
     raise ImproperlyConfigured("Error loading cx_Oracle module: %s" % e)
 
 # Some of these import cx_Oracle, so import them after checking if it's installed.
-from .client import DatabaseClient                          # NOQA isort:skip
-from .creation import DatabaseCreation                      # NOQA isort:skip
-from .features import DatabaseFeatures                      # NOQA isort:skip
-from .introspection import DatabaseIntrospection            # NOQA isort:skip
-from .operations import DatabaseOperations                  # NOQA isort:skip
-from .schema import DatabaseSchemaEditor                    # NOQA isort:skip
-from .utils import dsn, Oracle_datetime                     # NOQA isort:skip
-from .validation import DatabaseValidation                  # NOQA isort:skip
+from .client import DatabaseClient  # NOQA
+from .creation import DatabaseCreation  # NOQA
+from .features import DatabaseFeatures  # NOQA
+from .introspection import DatabaseIntrospection  # NOQA
+from .operations import DatabaseOperations  # NOQA
+from .schema import DatabaseSchemaEditor  # NOQA
+from .utils import Oracle_datetime, dsn  # NOQA
```

Comment:
I ran `isort -rc .` per the documentation manually, and it swapped the order of these.  If isort is run programatically i can revert those changes.

Reply:
> Can you tell me how do I open my issue for django I am new to django and github I have some questions about django

this may be help 
https://code.djangoproject.com